Added contributors to the list

Adriani90 opened the pull request:
### Link to issue number:
contributes to #9725. 

### Summary of the issue:
Some long term contributors are not yet in the contributors list and they asked for that on the translations maillist. Also many other contributors which are very active on users lists, addons maillist etc. are still not on the list.

### Description of how this pull request fixes the issue:
added the names to the contributors.txt 

### Testing performed:
Ran NVDA from local branch and checked that the list is updated

### Known issues with pull request:
none

### Change log entry:
== changes ==
- Updated the list of contributors.

Adriani90 committed:
Aded contributors to the list

Andre9642 commented:
I'd suggest to add the following link to the contributors.txt: <https://github.com/nvaccess/nvda/graphs/contributors>. Indeed, several gitHub contributors are not listed in `contributors.txt`, like:

* Christopher Toth
* Bill Dengler
* Aron Ocsvari
* Josiel Santos
* Karol Pecyna
* Blake Oliver
* Robert Haenggi
* Mikolaj Holysz
* André-Abush Clause
* Etc.

Adriani90 committed:
Added the link to the grafics of contributors on Github and added new names to the list

Adriani90 commented:
@Andre9642 I have added the link to the list along with a short description at the top of the list and added the names mentioned by you at the bottom of the list. In my view, I think we will never be able to include really everyone to the list because we do not know really how many people contribute to NVDA out there that we don't know of. So I think the best way is to raise such a pull request from time to time and add new names. As of now I can add as many names as possible. If someone knows further people who should be included on this list, please comment on this pull request.

michaelDCurran commented:
I will merge this pr to beta this Friday after 0 UTC. This gives a few more days for people to get their names added if missing. Then, they will be included in rc1.

lukaszgo1 commented on contributors.txt:
```diff
@@ -1,6 +1,11 @@
-﻿NV Access Limited
+﻿This is a list of people and organizations which contributed to the NVDA project in either way since the project has started. For an overview of code contributors, see also
```
I don't believe 'either way' is a proper phrase here because it implies that it is possible to contribute only in two ways.

josephsl commented:
Hi, I think a good word for this could be “various”. Thanks.

 

From: Łukasz Golonka <notifications@github.com> 
Sent: Wednesday, June 19, 2019 11:45 AM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: Re: [nvaccess/nvda] Added contributors to the list (#9727)

 

@lukaszgo1 commented on this pull request.

  _____  

In contributors.txt <https://github.com/nvaccess/nvda/pull/9727#discussion_r295454941> :

> @@ -1,6 +1,11 @@
 
-﻿NV Access Limited
 
+﻿This is a list of people and organizations which contributed to the NVDA project in either way since the project has started. For an overview of code contributors, see also
 

I don't believe 'either way' is a proper phrase here because it implies that it is possible to contribute only in two ways.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/9727?email_source=notifications&email_token=AB4AXEBELUIMIQ6KJNQADT3P3J5CPA5CNFSM4HXQICIKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB4B3SAY#pullrequestreview-251902211> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AB4AXEDOH52DS32DZA32OJTP3J5CPANCNFSM4HXQICIA> .

lukaszgo1 commented:
I don't know if they are any strict requirements  for being placed in the contributors file, but if not and my contributions are enough I'd like to ask to be added into it.

Adriani90 committed:
Added more names to the list

Adriani90 commented:
Lukasz ofcourse in my view you contribute alot to this project. I have added your name as well.

Adriani90 committed:
Addedd another bunch of names to the list

Adriani90 committed:
fixed a small spelling error

XLTechie commented:
Why is any phrase with "way" in it necessary? Why not just:

"
This is a list of people and organizations which have contributed to the NVDA 
project since the project has started. For an overview of code contributors, see 
also
"

I would also suggest changing "which" to "who"; and "has started" to "began" or 
at least just "started", but those were not what was asked.:)

Adriani90 committed:
Rewritten the first sentence and added some new names to the list.

Adriani90 committed:
Added Doug Lee to the list

Adriani90 commented:
I decided to add also a line in changes.t2t so people are aware that there has been an update to the list. @michaelDCurran as of now the commit for changes is for 2019.3. This is just one string. Should I amend it and commit it to your RC branch?

lukaszgo1 commented:
I'm afraid change log entry isn't a good idea here because it would be merged to beta directly, and the addition to change log wouldn't be translated.

Adriani90 commented:
Not necessarily, in case Mick decides to release a 2019.2.1 this could be translated. Or am I wrong?
Anyway, in my view it is important to let people know that there is a list of contributors and that it is updated from time to time. If there will not be a 2019.2.1, then I suggest to implement this in 2019.3.

josephsl reviewed (changes requested):
Hi,

I would like to request not editing What's New document in the future. Although mentioning new contributors has its merits, it isn't necessary and could open up issues requesting trivial changes be documented. For these, git log would suffice for now.

Thanks.

Adriani90 commented:
Oh yes I just discovered editing changes is not a really good idea :-). I have reverted the coresponding commit.

michaelDCurran commented:
I have rebased the branch and retargeted pr the to beta before merging.
